Tests/Http/Connection/HttpConnectionSessionsTests.swift

```diff
@@ -5,16 +5,15 @@
 @testable import MobileCoin
 import XCTest
 
-
-class HttpConnectionSessionTests: XCTestCase {
+class HttpConnectionSessionsTests: XCTestCase {
```

Comment:
Did you mean to pluralize Sessions?
```suggestion
class HttpConnectionSessionTests: XCTestCase {
```

Reply:
needs to match the filename